src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -65,6 +86,32 @@ export const PupilViewsReview = (props: PupilViewsReviewProps) => {
     </OakLessonBottomNav>
   );
 
+  const handleShareResultsClick = async () => {
```

Comment:
Share is a confusing name here as what you're actually doing is generating printable results. Can we change all the references to 'share' to 'Printable' so this function would be renamed 'handlePrintableResultsClick'